deploy/kubernetes/base/controller.yaml

```diff
@@ -65,6 +64,9 @@ spec:
         - name: node-update-controller
           image: registry.k8s.io/cloud-provider-ibm/ibm-powervs-block-csi-driver:main
           command: ["/node-update-controller"]
+          env:
+            - name: API_KEY_PATH
+              value: /etc/secrets/IBMCLOUD_API_KEY
```

Comment:
Is there any reason for adding the key to this controller?

Reply:
After looking unto the code, it looks like, the node update controller also needs the apikey (it is also initialising the authenticator), this apikey is needed to GetPVMInstancedetails. Hence, added here too